src/main/java/uk/gov/hmcts/reform/ccd/client/model/CaseDetails.java

```diff
@@ -13,19 +14,25 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CaseDetails {
 
+    @JsonProperty("id")
     private Long id;
 
+    @JsonProperty("jurisdiction")
```

Comment:
why is this added?

Reply:
does not hurt .. as in consistent with the other properties.

Reply:
its unnecessary, it hurts imo adds more cruft than is needed, json property is only needed when remapping properties, please remove